scripts/jest/jest-cli.js

```diff
@@ -229,12 +229,12 @@ function validateOptions() {
 function getCommandArgs() {
   // Add the correct Jest config.
   const args = ['./scripts/jest/jest.js', '--config'];
-  if (argv.build) {
+  if (argv.project === 'devtools') {
```

Comment:
DevTools *always* uses the `build` config, but not vice versa, so it needs to come first.

Having one if/else condition flow for two flags seems a bit fragile, but I didn't want to make a larger change here.

Reply:
Yeah that makes sense.